docs/apache-airflow-providers-influxdb/connections/influxdb.rst

```diff
@@ -31,17 +31,12 @@ Extra (required)
     Specify the extra parameters (as json dictionary) that can be used in InfluxDB
     connection.
 
-    ``token``: (required) `Create token <https://docs.influxdata.com/influxdb/cloud/security/tokens/create-token/>`_
-    using the influxdb cli or UI
-
-    ``org_name``: (required) `Create org <https://docs.influxdata.com/influxdb/cloud/reference/cli/influx/org/create/>`_
```

Comment:
If token and org name are required lets customize the Ui form for to have them. Its much nicer than passing them as kwargs

Reply:
@romsharon98 i think we shouldnt remove this but just mention its not defined via extra. 
This is what we do in other providers as well 
https://airflow.apache.org/docs/apache-airflow-providers-google/stable/connections/bigquery.html#configuring-the-connection